packages/gatsby-transformer-mdx-introspection/gatsby-node.js

```diff
@@ -54,7 +49,12 @@ async function introspectMdx({
         }
       });
     })
-    .process(node.rawBody);
+    .process(node.rawBody, (error, file) => {
+      if (error) {
+        console.log(String(file));
+        throw error;
```

Comment:
We need to handle this error otherwise the error it's swallowed and the process exits without any log.